packages/shared/src/utils/mounted/mounted.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+export const useIsMounted = () => {
+    const is_mounted = React.useRef(false);
+
+    React.useEffect(() => {
+        is_mounted.current = true;
+
+        return () => (is_mounted.current = false);
+    }, []);
+    return is_mounted;
+};
```

Comment:
Can we create a `hooks` folder and move this file there?

Reply:
Done, thank you Bala! :)